src/libraries/System.Formats.Cbor/src/System/Formats/Cbor/Reader/CborReader.Array.cs

```diff
@@ -8,11 +8,11 @@ public partial class CborReader
         /// <summary>Reads the next data item as the start of an array (major type 4).</summary>
         /// <returns>The length of the definite-length array, or <see langword="null" /> if the array is indefinite-length.</returns>
         /// <exception cref="InvalidOperationException">The next data item does not have the correct major type.</exception>
-        /// <exception cref="CborContentException">The next value has an invalid CBOR encoding.
-        /// -or-
-        /// There was an unexpected end of CBOR encoding data.
-        /// -or-
-        /// The next value uses a CBOR encoding that is not valid under the current conformance mode.</exception>
+        /// <exception cref="CborContentException"><p>The next value has an invalid CBOR encoding.</p>
+        /// <p>-or-</p>
```

Comment:
cc @RussKie who will help fix the backporting tool.

We need to make sure that exceptions get formatted like this when backporting them from MS Docs to triple slash, so that line breaks get respected.